terraform/modules/provision-k8s/k8s-network-policy.tf

```diff
@@ -47,10 +47,13 @@ resource "kubernetes_network_policy" "default-deny" {
 
 resource "helm_release" "calico" {
   name       = "calico"
-  namespace  = "calico-system"
+  namespace  = kubernetes_namespace.calico-system.id
```

Comment:
I was trying to figure out how to do this when I did this, but I'm glad you fixed this haha..